### PR TITLE
project in process 

### PR DESCRIPTION
**Pull request by ViktorTsupa**
- [DEMO LINK](https://ViktorTsupa.github.io/layout_stars/)
- [TEST REPORT LINK](https://ViktorTsupa.github.io/layout_stars/report/html_report/)

### CODE REVIEW

**Comment by ViktorTsupa** [comment]
- [DEMO LINK](https://ViktorTsupa.github.io/layout_stars/)
- [TEST REPORT LINK](https://ViktorTsupa.github.io/layout_stars/report/html_report/)

- [x] Yellow stars are added with container modifier + pseudo-selector (NO extra classes)
- [x] Each BEM block has its own separate file
- [x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

**Comment by ViktorTsupa** [comment]
edited according to your comments, please review again, thank you 😃

File: src/style.css [hunk 1]
```diff
@@ -1 +1,65 @@
...
+  content: "";
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  position: absolute;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--2 li:nth-child(-1n + 2)::after {
```
Comment (anastasiiavorobiova) [inline src/style.css:27]:
```suggestion
.stars--2 li:nth-child(-1n + 2)::after,
... {
```
Consider using commas to group your selectors and avoid repeating code

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

**Review by VitaliyBondarenko1982 — APPROVED** [review]

**Pull request closed without merging**